src/index.html

```diff
@@ -1,12 +1,375 @@
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#" class="header__logo-link">
+          <img
+            class="header__logo-img"
+            src="./images/logo.png"
+            alt="Logo image"
+          >
+        </a>
+      </div>
+
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Смартфоны и телефоны
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Гаджеты
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Планшеты
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              ФОТО
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              ВИДЕО
+            </a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="products">
+        <div class="card products__card" data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="Product image"
+          >
+          <h2 class="card__heading">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
+          <p class="card__code">Код товара: 195434</p>
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+            </div>
+            <div class="card__rating-count">Отзывов: 5</div>
+          </div>
+          <div class="card__price">
+            <p class="card__price-text">Цена:</p>
+            <p class="card__price-amount">
+              <strong>
+                69 999 грн
+              </strong>
+            </p>
+          </div>
+          <button
+            class="card__btn"
+            data-qa="hover"
```

Comment:
```suggestion
            data-qa="card-hover"
```
add data-qa="card-hover" (not just hover) to the link Купить inside the first card